<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reports</class>
 <widget class="QMainWindow" name="reports">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Reports</string>
  </property>
  <property name="styleSheet">
   <string>QWidget {
    background-color: #f5f7fa;
    font-family: &quot;Segoe UI&quot;, &quot;Helvetica Neue&quot;, Arial, sans-serif;
    font-size: 14px;
    color: #000000;
}
QLabel {
    color: #000000;
}
QLabel#reportsPageTitleLabel {
    font-size: 24px;
    font-weight: 600;
    color: #1a1a1a;
}
QTableView {
    background-color: #ffffff;
    border: 1px solid #e2e8f0;
    border-radius: 6px;
    gridline-color: #e5e7eb;
}
QTableView::item {
    padding: 8px;
}
QTableView::item:selected {
    background-color: #e5e7eb;
    color: #1f2937;
}
QPushButton {
    background-color: #2563eb;
    color: white;
    border-radius: 6px;
    padding: 12px 24px;
    min-height: 48px;
    font-weight: 500;
    font-size: 15px;
}
QPushButton:hover {
    background-color: #1d4ed8;
}
QPushButton:pressed {
    background-color: #1e40af;
}
QPushButton#backToDashboardButton {
    background-color: #e5e7eb;
    color: #1a1a1a;
}
QPushButton#backToDashboardButton:hover {
    background-color: #d1d5db;
}
QLabel#totalNotesLabel {
    color: #2563eb;
    font-weight: 600;
    font-size: 16px;
}
QComboBox {
    background-color: #ffffff;
    border: 1px solid #e2e8f0;
    border-radius: 6px;
    padding: 8px 12px;
    min-width: 200px;
    color: #000000;
}
QComboBox:focus {
    border: 2px solid #2563eb;
    outline: none;
}
QComboBox::drop-down {
    border: none;
    width: 20px;
}
QComboBox::down-arrow {
    image: none;
    border: none;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>24</number>
    </property>
    <property name="leftMargin">
     <number>32</number>
    </property>
    <property name="topMargin">
     <number>32</number>
    </property>
    <property name="rightMargin">
     <number>32</number>
    </property>
    <property name="bottomMargin">
     <number>32</number>
    </property>
    <item>
     <widget class="QLabel" name="reportsPageTitleLabel">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>-1</pointsize>
        <weight>600</weight>
       </font>
      </property>
      <property name="text">
       <string>Reports</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>16</number>
       </property>
       <item>
        <widget class="QLabel" name="totalNotesLabel">
         <property name="text">
          <string>Total Notes: 0</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="reportTypeLabel">
         <property name="text">
          <string>Report Type:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="reportTypeComboBox">
         <item>
          <property name="text">
           <string>Total Notes Per User</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Most Accessed Notes</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="generateReportButton">
         <property name="text">
          <string>Generate</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="reportTableView">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <property name="spacing">
       <number>16</number>
      </property>
      <item>
       <widget class="QPushButton" name="exportReportButton">
        <property name="text">
         <string>Export Report</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="backToDashboardButton">
        <property name="text">
         <string>Back to Dashboard</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>

